src/ClusterBootstrap/deploy.py

```diff
@@ -3577,6 +3628,17 @@ def run_command( args, command, nargs, parser ):
             print "Error: kubernetes need a subcommand."
             exit()
 
+    elif command == "gpulabel":
+        kubernetes_label_GpuTypes()
+
+    elif command == "gen_scripts":
+        # print(config["azure_cluster"].keys())
+        gen_dns_config_script()
+        gen_pass_secret_script()
+
+    elif command == "setconfigmap":
```

Comment:
This may not work. we cannot assume kubectl is working on dev machine.

Reply:
Di told me he edited the jobmanager, so it's not necessary to run this. I haven't tested yet. If it works, I'd delete this component, otherwise we can discuss next meeting.

Reply:
Let's delete it.